### PR TITLE
[ASM] - Expander - Update service owner playbook

### PR DESCRIPTION
**Pull request by BigEasyJ**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
#25927

## Description
- Updates Pack ReadMe
- Fixes an issue where service owner were not found.
- Updates the Cortex ASM - Service Ownership playbook to check for GCP IAM integration and for the existence of service owners.

## Screenshots
Cortex_ASM_-_Service_Ownership

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by BigEasyJ** [comment]
cc: @johnnywilkes @capanw @kball-pa @ptoman-pa

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_9.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Playbooks
+
+##### Cortex ASM - Service Ownership
+
+- Fixed an issue where service owner were not found.
```
Comment (kball-pa) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_6_9.md:6]: nit: typo, "service owners"

**Review by kball-pa — APPROVED** [review]
LGTM

**Review by johnnywilkes — APPROVED** [review]
@BigEasyJ , looked this over and it is good to go!

**Comment by johnnywilkes** [comment]
@samuelFain, this is an Xpanse only pack change and we have approved internally.  Please merge when possible.

**Review by samuelFain — APPROVED** [review]

**Pull request merged**